source/Cosmos.IL2CPU/AppAssembler.cs

```diff
@@ -1371,7 +1371,7 @@ public void EmitEntrypoint(MethodBase aEntrypoint, MethodBase[] aBootEntries = n
             }
             else
             {
-                X86.IL.Call.DoExecute(Assembler, null, aEntrypoint.DeclaringType.BaseType.GetMethod(CompilerEngine.UseGen3Kernel ? "EntryPoint" : "Start"), null, xCurLabel, CosmosAssembler.EntryPointName + ".AfterStart", DebugEnabled);
```

Comment:
Why did you change this?

Reply:
To override Kernel.Start

Reply:
I see, that is a sensible change. Can you add a sentence to https://github.com/CosmosOS/Cosmos/blob/master/Docs/articles/Kernel/Startup.md/ so that people are aware of the possibility?